### PR TITLE
Update for EDL

### PR DESCRIPTION
**Pull request by ShirleyDenkberg**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
Update descriptions for EDL. Content packs included are:
SimpleAPIProxy
PublishList
PingCastle

## Description
A few sentences describing the overall goals of the pull request's commits.

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by ShirleyDenkberg** [comment]
Needs a Readme to be generated for:
- PublishList
- SimpleAPIProxy

File: Packs/PingCastle/Integrations/PingCastle/PingCastle.yml [hunk 1]
```diff
@@ -13,7 +13,8 @@ configuration:
   name: longRunning
   type: 8
   required: false
-- display: Listen port, e.g. 7000
+- additionalinfo: Runs the service on this port from within Cortex XSOAR. Requires a unique port for each long-running integration instance. Do not use the same port for multiple instances. <br>Note: If you click the test button more than once, a failure may occur mistakenly indicating that the port is already in use. (For Cortex XSOAR 8 and Cortex XSIAM) If you do not enter a Listen Port, an unused port for the EDL will automatically be generated when the instance is saved. However, if using an engine, you must enter a Listen Port.
```
Comment (ShirleyDenkberg) [inline Packs/PingCastle/Integrations/PingCastle/PingCastle.yml:16]:
```suggestion
- additionalinfo: Runs the service on this port from within Cortex XSOAR. Requires a unique port for each long-running integration instance. Do not use the same port for multiple instances. Note: If you click the test button more than once, a failure may occur mistakenly indicating that the port is already in use. (For Cortex XSOAR 8 and Cortex XSIAM) If you do not enter a Listen Port, an unused port for Ping Castle will automatically be generated when the instance is saved. However, if using an engine, you must enter a Listen Port. 
```

File: Packs/PingCastle/Integrations/PingCastle/README.md [hunk 2]
```diff
@@ -8,9 +8,9 @@ This integration was integrated and tested with version 6.0.0 of PingCastle
 
     | **Parameter** | **Description** | **Required** |
     | --- | --- | --- |
-    | API Key | The API Key PingCastle must use to send reports | True |
-    | Long running instance | Whether this instance should listen for reports | False |
-    | Listen port, e.g. 7000 |  | False |
+    | API Key | The API Key PingCastle must use to send reports. | True |
+    | Long running instance | Whether this instance should listen for reports. | False |
+    | Listen port, e.g. 7000 | Runs the service on this port from within Cortex XSOAR. Requires a unique port for each long-running integration instance. Do not use the same port for multiple instances. <br>Note: If you click the test button more than once, a failure may occur mistakenly indicating that the port is already in use.  <br> (For Cortex XSOAR 8 and Cortex XSIAM) If you do not enter a Listen Port, an unused port for the EDL will automatically be generated when the instance is saved. However, if using an engine, you must enter a Listen Port.  | False |
```
Comment (ShirleyDenkberg) [inline Packs/PingCastle/Integrations/PingCastle/README.md:13]:
```suggestion
    | Listen port, e.g. 7000 | Runs the service on this port from within Cortex XSOAR. Requires a unique port for each long-running integration instance. Do not use the same port for multiple instances. <br>Note: If you click the test button more than once, a failure may occur mistakenly indicating that the port is already in use.  <br> (For Cortex XSOAR 8 and Cortex XSIAM) If you do not enter a Listen Port, an unused port for Ping Castle will automatically be generated when the instance is saved. However, if using an engine, you must enter a Listen Port.  | False |
```

File: Packs/PublishList/Integrations/PublishList/PublishList.yml [hunk 3]
```diff
@@ -9,7 +9,7 @@ configuration:
   name: longRunning
   type: 8
   required: false
-- additionalinfo: Runs the web server on this port from within Cortex XSOAR. Requires a unique port for each long-running integration instance. Do not use the same port for multiple instances.
+- additionalinfo: Runs the service on this port from within Cortex XSOAR. Requires a unique port for each long-running integration instance. Do not use the same port for multiple instances. Note: If you click the test button more than once, a failure may occur mistakenly indicating that the port is already in use. (For Cortex XSOAR 8 and Cortex XSIAM) If you do not enter a Listen Port, an unused port for the EDL will automatically be generated when the instance is saved. However, if using an engine, you must enter a Listen Port.
```
Comment (ShirleyDenkberg) [inline Packs/PublishList/Integrations/PublishList/PublishList.yml:12]:
```suggestion
- additionalinfo: Runs the service on this port from within Cortex XSOAR. Requires a unique port for each long-running integration instance. Do not use the same port for multiple instances. Note: If you click the test button more than once, a failure may occur mistakenly indicating that the port is already in use. (For Cortex XSOAR 8 and Cortex XSIAM) If you do not enter a Listen Port, an unused port for Publish List will automatically be generated when the instance is saved. However, if using an engine, you must enter a Listen Port.
```

File: Packs/SimpleAPIProxy/Integrations/SimpleAPIProxy/SimpleAPIProxy.yml [hunk 4]
```diff
@@ -9,7 +9,7 @@ configuration:
   name: longRunning
   type: 8
   required: false
-- additionalinfo: Runs the web server on this port from within Cortex XSOAR. Requires a unique port for each long-running integration instance. Do not use the same port for multiple instances.
+- additionalinfo: Runs the web server on this port from within Cortex XSOAR. Requires a unique port for each long-running integration instance. Do not use the same port for multiple instances.Note: If you click the test button more than once, a failure may occur mistakenly indicating that the port is already in use. (For Cortex XSOAR 8 and Cortex XSIAM) If you do not enter a Listen Port, an unused port for the EDL will automatically be generated when the instance is saved. However, if using an engine, you must enter a Listen Port.
```
Comment (ShirleyDenkberg) [inline Packs/SimpleAPIProxy/Integrations/SimpleAPIProxy/SimpleAPIProxy.yml:12]:
```suggestion
- additionalinfo: Runs the web server on this port from within Cortex XSOAR. Requires a unique port for each long-running integration instance. Do not use the same port for multiple instances. Note: If you click the test button more than once, a failure may occur mistakenly indicating that the port is already in use. (For Cortex XSOAR 8 and Cortex XSIAM) If you do not enter a Listen Port, an unused port for Simple API Proxy will automatically be generated when the instance is saved. However, if using an engine, you must enter a Listen Port.
```

**Review by ShahafBenYakir — APPROVED** [review]

**Pull request merged**